### PR TITLE
Update kinto admin

### PR DESCRIPTION
**Pull request by Natim**
f? @n1k0

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,7 +44,12 @@ build-requirements:
 update-kinto-admin:
 	rm -fr kinto_admin/static/*
 	npm install -g kinto-admin
-	kinto-admin build -d kinto_admin/static/
+	rm -fr tmp
+	git clone https://github.com/Kinto/kinto-admin.git tmp
+	cd tmp; git checkout v$$(kinto-admin --version); npm install
```
Comment (n1k0) [inline Makefile]: I'm confused as to why we still need the kinto-admin package installed from npm? Now we're cloning the repo, we could simply checkout the tag and build from there. That would also allow specifying the version explicitly which would prevent possible *surprises*.

**Comment by n1k0** [comment]
LGTM though I think we don't need to install the kinto-admin package from npm anymore :)

File: Makefile (hunk 1 above)
Comment (Natim) [inline Makefile]: My goal was to install directly the last released version of kinto-admin.

Comment (n1k0) [inline Makefile]: `git tag|sort -r|head -n 1`?

Comment (Natim) [inline Makefile]: ``git tag | tail -n 1``

Comment (n1k0) [inline Makefile]: hmm not sure the tags are consistently listed in the desired order, sorting by version seems safer.

Comment (Natim) [inline Makefile]: According to the doc, the output from git tag is ordered alphabetically.

Comment (n1k0) [inline Makefile]: ok then

**Pull request merged**